dynamo/configuration.py

```diff
@@ -137,6 +140,35 @@ def allowed_X_layer_names():
     def init_uns_pp_namespace(adata: AnnData):
         adata.uns[DynamoAdataKeyManager.UNS_PP_KEY] = {}
 
+    def get_excluded_layers(
+        X_total_layers: bool = False,
+        splicing_total_layers: bool = False
+    ):
+        """ Get a list of excluded layers based on the provided arguments."""
+        excluded_layers = []
+        if not X_total_layers:
+            excluded_layers.extend(["X"])
+        if not splicing_total_layers:
+            excluded_layers.extend(["spliced", "unspliced"])
+        return excluded_layers
```

Comment:
can you document the meaning of each argument? 
"""Get a list of excluded layers based on the provided arguments.""" (remove the space before Get)